File: 3001-3250/3227. Vowels Game in a String/solution.java
class Solution {
    public boolean doesAliceWin(String s) {
        for( char c : s.toCharArray() ){
            switch(c){
                case 'a' , 'e' , 'u' , 'i' , 'o' : 
                    return true  ;
            }
        }
        return false ; 
    }
}

File: 3001-3250/3227. Vowels Game in a String/solution.rs
impl Solution {
    pub fn does_alice_win(s: String) -> bool {
        for ch in s.chars() {
            if( ch == 'a' || ch == 'e' || ch == 'i' || ch == 'u' || ch == 'o' ){
                return true ; 
            }
        }
        return false ; 
    }
}